### ISSUE TITLE
Problem with Notation

### ISSUE THREAD

**Issue by jyotman** [onury/accesscontrol#19]
```js
accessControl.grant('user').create('project', ['*', '!approved']);
accessControl.grant('admin').extend('user').create('project');

accessControl.can('user').create('project').attributes; // [ '*', '!approved' ] ✔️
accessControl.can('admin').create('project').attributes; // [ '*', '!approved' ] ❌
accessControl.can(['user', 'admin']).create('project').attributes; // [ '*' ] ✔️
```

As you can see, the last two lines should have produces the same results - `[ '*' ]`. So i debugged the code and found that the problem seems to be with the Notation module that AccessControl uses.

```js
notation.Glob.union(['*', '!approved'], ['*']); // [ '*' ]
notation.Glob.union(['*'], ['*', '!approved']) // [ '*', '!approved' ]
```

Then I also saw that you might've fixed the [issue](https://github.com/onury/notation/issues/3) in Notation. Am I right in assuming that this bug will be fixed when it start using the latest Notation module version? :1st_place_medal:

**Comment by onury** [comment]
Yes @jyotman, I've fixed it in dep. module `Notation`. Also improved some other things.

I'm working on AccessControl right now. Expect a new release soon.
Thanks.

**Comment by onury** [comment]
Fixed in v2.0.0
